Dialogue/Mid-Mission/ii_boss_ahl_nyau.csv

```diff
@@ -1,12 +1,12 @@
 IncidentName#0,"""Clash with Nyau!"""
 TrialBeginMsg#0,"""Beat Nyau!"""
 TrialAbstract#0,"""Beat $(0)"""
-NpcComOnBegin#0,"""ニャウの出現を確認しました。<br>面倒かと思われますが、各員<br>適切な対処をお願いします。"""
-NpcComOnSuccess#0,"""ニャウ、ようやく撤退するようです。<br>対応、感謝します。"""
-NpcComOnBegin_Other#0,"""ニャウの出現を確認しました。<br>面倒かと思われますが、各員<br>適切な対処をお願いします。"""
-NpcComOnSuccess_Other#0,"""ニャウ、ようやく撤退するようです。<br>対応、感謝します。"""
-NpcComOnBegin_Darker#0,"""ニャウの出現を確認しました。<br>面倒かと思われますが、各員<br>適切な対処をお願いします。"""
-NpcComOnSuccess_Darker#0,"""ニャウ、ようやく撤退するようです。<br>対応、感謝します。"""
+NpcComOnBegin#0,"""I've confirmed Nyau's appearance who may be troublesome. Everyone, please take appropriate action."""
```

Comment:
I'd split this first sentence a bit, something like "I've confirmed Nyau's appearance. It may be troublesome."